packages/astro/src/astro-components/unstyled/SignInButton.astro

```diff
@@ -3,9 +3,9 @@ import type { HTMLTag, Polymorphic } from 'astro/types'
 import type { SignInProps } from "@clerk/types";
 type Props<Tag extends HTMLTag = HTMLTag> = Polymorphic<SignInProps & { as: Tag; mode?: 'redirect' | 'modal' }>
 
-import { customAlphabet, urlAlphabet } from "nanoid";
+import { generateSafeId } from '@clerk/astro/internal';
```

Comment:
❓ Since this import is already inside the astro package why do we use the `'@clerk/astro/internal';` instead of relative path?

Reply:
This is a very nice question. Anything that is inside `astro-components/*` will be just copied to `components/*` and will be accessible by `@clerk/astro/components/*`.

Because there is no building step here, the relative paths will not be resolvable by the host application.